lib/spark_api/models/floplan.rb

```diff
@@ -0,0 +1,25 @@
+module SparkApi
+  module Models
+    class FloPlan < Base
+      extend Subresource
+      self.element_name = 'floplans'
+
+      def initialize(attributes={})
+        unless attributes['Images'].nil?
+          images = []
+          thumbnails = []
+          attributes['Images'].each do |img|
+            if img["Type"].include?('thumbnail')
+              thumbnails << img
+            else
+              images << img
+            end
+          end
+          attributes['Images'] = images
+          attributes['Thumbnails'] = thumbnails
```

Comment:
To speak to convention a bit, when we've created convenience methods in this gem, they almost always follow ruby convention for naming. For example, take a look at how the Account model enhances the `Images` array:
https://github.com/sparkapi/spark_api/blob/4c8200e4d7ff500defb1a7080d1c955aa8a6f5dc/lib/spark_api/models/account.rb#L15-L17

That kind of pattern may be useful here, and be more in keeping with the way we've telegraphed when something is an API attribute (CameCase) vs something else (lower_case, aka proper ruby convention)